### ISSUE TITLE
name change ?

### ISSUE THREAD

**Issue by jm-pro** [satyajitghana/PySodium#1]
Hi
sodium is a very famous cryptographic library, cf https://libsodium.gitbook.io/doc/bindings_for_other_languages
There are a few python wrappers for it already, one of them being https://pypi.org/project/pysodium/ 
you might want to consider a new name!

**Comment by satyajitghana** [comment]
Yes, I am also considering rewriting the library, but as of now there's no further development for this.
